### PR TITLE
chore(ci): type-check third-party Deno bots

### PR DESCRIPTION
**Pull request by wojpawlik**

### CODE REVIEW

File: .github/workflows/third_party.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                deno-version: v1.x
+
+            - name: Create import_map.json
+              run: |
+                set -o pipefail;
+                curl --silent https://cdn.deno.land/grammy/meta/versions.json |
+                jq '
+                    (.latest / ".")[0] as $major
+                    | .versions
+                    | map(select(startswith("\($major).")) | {
```
Comment (wojpawlik) [inline .github/workflows/third_party.yml:42]:
```suggestion
                    | map(select(startswith($major + ".")) | {
```
is probably more readable

Comment (roj1512) [reply .github/workflows/third_party.yml:42]: and less Swift-ish. Nice.

File: .github/workflows/third_party.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: Third-party bots
```
Comment (KnorpelSenf) [inline .github/workflows/third_party.yml:1]:
```suggestion
name: Type-check third-party bots
```

**Review by KnorpelSenf — APPROVED** [review]
LGTM, thank you so much!

**Pull request merged**